test/k8s-integration/cluster.go

```diff
@@ -150,9 +150,19 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string) err
 			return err
 		}
 	}
-	cmd := exec.Command("gcloud", "container", "clusters", "create", gkeTestClusterName,
-		locationArg, locationVal, "--cluster-version", *gkeClusterVer, "--num-nodes", strconv.Itoa(numNodes),
-		"--quiet", "--machine-type", "n1-standard-2", "--image-type", imageType)
+
+	var cmd *exec.Cmd
+	if useManagedDriver {
+		// PD CSI Driver add on is enabled only in gcloud beta.
+		cmd = exec.Command("gcloud", "beta", "container", "clusters", "create", gkeTestClusterName,
+			locationArg, locationVal, "--cluster-version", *gkeClusterVer, "--num-nodes", strconv.Itoa(numNodes),
```

Comment:
I wonder if we're going to need an option for release channels instead of just specifying cluster-version. For example, now the rapid channel is the only way to get 1.17; the highest --cluster-version one can pass is 1.16.8.

Reply:
Ack, have added support for release channels.